week-3/todo-list/script.js

```diff
@@ -1,25 +1,68 @@
-function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+let todos = [];
+const todoList = document.getElementById("todo-list");
+const removeAllButton = document.getElementById("remove-all-completed");
+
+function addNewTodo() {
+  let input = document.querySelector("input").value;
+  todos[todos.length] = { task: input, completed: false };
+  populateTodoList();
```

Comment:
Once I click on the "Add todo"button, the input field is not cleared. Which means I risk adding the same todo multiple times. How could you address this to make for a better user experience?